<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>515</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Parse radar station specs</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>450</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Parse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="freqLowerB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>lower</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="freqUpperB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>upper</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Band</string>
    </property>
   </widget>
   <widget class="QComboBox" name="freqUnitBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GHz</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pulse power (Wt)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pulsePower">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pulseLength">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>min</string>
    </property>
   </widget>
   <widget class="QComboBox" name="timeUnitBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ms</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>us</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ns</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pulse length</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Antenna Gain (dB)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="antennaGain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="Sql">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>281</width>
      <height>381</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextEditable|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SQL query</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Country">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Country</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Azimuth">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Azimuth coverage (°)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Elev angle coverage (°)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EAng">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radar name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RadarName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QComboBox" name="polarizationTypeBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Unknown</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Linear horizontal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Linear vertical</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Circular</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Polarization type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="siBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>10³</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10⁶</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10⁹</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="pulseLength_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>max</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pulsePeriod_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pulse period</string>
    </property>
   </widget>
   <widget class="QComboBox" name="timeUnitBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ms</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>us</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ns</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="pulsePeriod">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>min</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="UTF8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>450</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>UTF-8</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="Windows1251">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>450</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Windows 1251</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Encoding</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rotation rate (rpm)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RotationRate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Beamwidth (°)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BeamWidth">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>vert. min</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pulsePower_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>max</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BeamWidth_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>horiz. min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Scan sector (°)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ScanSector_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>horiz.min</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ScanSector">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>vert. min</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BeamWidth_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>horiz. max</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BeamWidth_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>vert. max</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ScanSector_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>vert. max</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ScanSector_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>horiz. max</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>425</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>set style</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>RadarName</tabstop>
  <tabstop>Country</tabstop>
  <tabstop>freqLowerB</tabstop>
  <tabstop>freqUpperB</tabstop>
  <tabstop>freqUnitBox</tabstop>
  <tabstop>pulsePower</tabstop>
  <tabstop>pulsePower_2</tabstop>
  <tabstop>siBox</tabstop>
  <tabstop>antennaGain</tabstop>
  <tabstop>pulseLength</tabstop>
  <tabstop>pulseLength_2</tabstop>
  <tabstop>timeUnitBox</tabstop>
  <tabstop>Azimuth</tabstop>
  <tabstop>EAng</tabstop>
  <tabstop>pulsePeriod</tabstop>
  <tabstop>pulsePeriod_2</tabstop>
  <tabstop>timeUnitBox_2</tabstop>
  <tabstop>RotationRate</tabstop>
  <tabstop>BeamWidth</tabstop>
  <tabstop>BeamWidth_2</tabstop>
  <tabstop>BeamWidth_3</tabstop>
  <tabstop>BeamWidth_4</tabstop>
  <tabstop>ScanSector</tabstop>
  <tabstop>ScanSector_2</tabstop>
  <tabstop>ScanSector_3</tabstop>
  <tabstop>ScanSector_4</tabstop>
  <tabstop>polarizationTypeBox</tabstop>
  <tabstop>UTF8</tabstop>
  <tabstop>Windows1251</tabstop>
  <tabstop>Sql</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
